examples/quad/main.rs

```diff
@@ -698,18 +762,21 @@ fn main() {
             cmd_buffer.finish();
 
             let submission = Submission {
-                command_buffers: Some(&cmd_buffer),
-                wait_semaphores: Some((&frame_semaphore, PipelineStage::BOTTOM_OF_PIPE)),
-                signal_semaphores: &[],
+                command_buffers: Some(&*cmd_buffer),
+                wait_semaphores: Some((
+                    &image_acquire_semaphores[frame],
+                    PipelineStage::COLOR_ATTACHMENT_OUTPUT,
+                )),
+                signal_semaphores: Some(&submission_complete_semaphores[frame]),
```

Comment:
I don't think this signal is really needed, given that we are presenting on the same queue

Reply:
It's needed. Presentation engine is abstracted by vulkan but requires synchronization with the presenting queue.

Reply:
uh oh, thanks for correction!

Reply:
For reference beside the specs: https://github.com/KhronosGroup/Vulkan-Docs/wiki/Synchronization-Examples#combined-graphicspresent-queue